packages/@vue/cli-ui/README.md

```diff
@@ -1,15 +1,43 @@
 # @vue/cli-ui
 
-Start development version:
+### Local development
+
+Before starting a local cli-ui instance, 
+consider following the [contributing guide](https://github.com/vuejs/vue-cli/blob/dev/.github/CONTRIBUTING.md) 
+in order to download all required dependencies of vue-cli's packages.
+ 
+Just after, you should build `@vue/cli-ui-addon-webpack` by running:
+```bash
+cd ../cli-ui-addon-webpack
+yarn build
+cd -
+```
+
+Then you should start a local Apollo server:
+
+```
+yarn run apollo
```

Comment:
This needs to be run in the `cli-ui` folder (not `cli-ui-addon-webpack`).

Reply:
I though `cd -` (like it's done in https://github.com/vuejs/vue-cli/blob/dev/.github/CONTRIBUTING.md) was enough. :thinking: 

Anyway I'm gonna change it to be more clear